bin/test_travis.sh

```diff
@@ -119,9 +119,15 @@ fi
 if [[ "${TEST_SYMPY}" == "true" ]]; then
     cat << EOF | python
 print('Testing SYMPY, split ${SPLIT}')
-import sympy
-if not sympy.test(split='${SPLIT}'):
+import sympy, coverage
+cov = coverage.Coverage()
+cov.start()
+if not sympy.test(split='${SPLIT}', subprocess=False):
```

Comment:
Thanks. It isn't clear to me how to run coverage.py with the tests running in subprocesses. Maybe I need the coverage function calls to be deeper in the test code so it executes in the subprocess.